src/common/java/net/minecraftforge/gradle/common/util/runs/IntellijRunGenerator.java

```diff
@@ -198,7 +203,13 @@ protected Map<String, Document> createRunConfiguration(@Nonnull final Project pr
                         {
                             gradleTask.setAttribute("name", "Gradle.BeforeRunTask");
                             gradleTask.setAttribute("enabled", "true");
-                            gradleTask.setAttribute("tasks", project.getTasks().getByName("prepare" + Utils.capitalize(runConfig.getTaskName())).getPath());
+                            final List<String> tasks = new ArrayList<>();
+                            final boolean copyResources = mc.getCopyIDEResources().getOrElse(false);
+                            if (mc.getEnableIdeaPrepareRuns().getOrElse(true) || copyResources)
```

Comment:
I feel like I would prefer a `.convention(true)` on the original property (probably constructor of MinecraftExtension) rather than a getOrElse(true) everywhere it is referenced.